recipes/python-statuspage/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set url_version = "0.3-pre" %}
+{% set version = url_version.replace("-", "") %}
+
+package:
+  name: python-statuspage
+  version: {{ version }}
+
+source:
+  fn: python-statuspage-{{ version }}.tar.gz
+  url: https://github.com/jayfk/statuspage/archive/{{ url_version }}.tar.gz
+  sha256: 51f6de360b3fffe6a1eac1f9d3ee08ab65adef972d272634f4564b1815e50c8d
+  patches:
+    # Fix setuptools dependency.
+    # --------------------------
+    # A few requirements are needed by setuptools.
+    # In particular, `packaging` and `six`. Here
+    # we add them to PyInstaller's hidden imports
+    # to ensure they are packaged in the final
+    # bundle. See PR link below for more details.
+    #
+    # https://github.com/jayfk/statuspage/pull/52
+    #
+    - PR_52.diff
+
+build:
+  number: 0
+  skip: true  # [win or py3k]
```

Comment:
Had issues getting it to pick up libraries for Python 3.x mainly on Mac (only on Travis CI). As we don't need multiple Python builds, but only need one. I figured it would be simplest to pick the one that worked reliably. Will raise an issue in the feedstock and investigate further later.